draft-secure-credential-transfer.md

```diff
@@ -203,9 +203,56 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Provisioning Information Structure
+
+The Provisioning Information is the data transfered via the Relay Server between the Sender device and Receiver device. Each use case defines its own specalized Provisioning Information format, but all formats must at least ahead to the following structure.
+
+| Key    | Type   | Description 
+| ------ | ------ | --- 
+| format | String | The Provisioning Information format that the message follows. This is used by the Sender device and Receiver device to know how to parse the message.
+
+##### Provisioning Information Format
+
+Each Provisioning Information format must have the message structure defined in an external specification.
+
+| Format Type                      | Spec Link | Description 
+| -------------------------------- | --------- | ---
+| digitalwallet.carkey.ccc         | [Link]()  | A digital wallet Provisioning Information for sharing a car key that follows the Car Connectivity Consortium specification.
+| digitalwallet.authorizationToken | [Link]()  | A digital wallet Provisioning Information for sharing a generic pass that relies solely on an authorization token.
```

Comment:
It would be nice to have a reverse dns type consistent format for the format types.

digitalwallet.carkey.ccc
digitalwallet.generic.authorizationToken
digitalwallet.future.unknown